components/navigation.html

```diff
@@ -8,13 +8,20 @@
         <div class="left">
             <!-- Search button -->
             <div class="search-container close">
-                <input type="search" name="searchInp" onchange="sendSearch(this.value, 'search.html')" onkeyup="searchSitesWithResult(this.value, 'search_results', false, true),searchInSite(this.value)" id="search_box" />
+                <!-- <input type="search" name="searchInp" id="searchInp" onchange="sendSearch(this.value, 'search.html')" onkeyup="searchSitesWithResult(this.value, 'search_results', false, true),searchInSite(this.value)" id="search_box" /> -->
```

Comment:
why is this commented out?

Reply:
Because I'm testing out at the moment whether the event `onkeyup` works on mobile devices too. I think I had a problem with the searcher on my phone but not sure.